mern/server/utils/parse.js

```diff
@@ -1,42 +1,72 @@
-import QA from "../models/question_answer.js"
+import QA from "../models/question_answer.js";
 
-export default function parse(transcript){
-
-    const lst = []
-    for (var i = 0; i<transcript.length; i++){
-
-        lst.push(createQAs(transcript[i]))
-    }
-
-    return lst
+//Parses the transcript by looking through each 'list' of questions and creating question answer pairs
+//See trancript JSON formatting for more details
+export default function parse(transcript) {
+  const lst = [];
+  for (var i = 0; i < transcript.length; i++) {
+    createQAs(transcript[i]);
+    lst.push(initialQAID(transcript[i]));
+  }
+  return lst;
 }
 
-function createQAs(qlist){
-
-    const initial = qlist[0].id
-
-    //qlist is a list of JSON objects that exists as questions/answer pairs
-    for (var i = 0; i <qlist.length; i++){
-
-        createQA(qlist[i])
-    }
-
-    // Return the first item in the list which should be the top level question
-    return initial
+//Loops through all questions in the JSON object qlist, making them into objects in the databse
+export function createQAs(qlist) {
```

Comment:
It might be better (in terms of SOLID Principles) to split these functions into separate files

Reply:
True. Need to think about the best way to split these up